inference-engine/src/vpu/graph_transformer/src/stages/topk.cpp

```diff
@@ -161,7 +158,7 @@ void FrontEnd::parseTopK(const Model& model, const ie::CNNLayerPtr& _layer, cons
                                                StageType::TopK,
                                                layer, inputs, realOutputs);
 
-    stage->attrs().set<Dim>("axis", axis);
+    stage->attrs().set<int64_t>("axis", layer->axis + layer->axis < 0 ? numDims - 1 : 0);
```

Comment:
Use brackets here to make it clear which operation will be evaluated first (`+` or `?`)

Reply:
Moreover, I think it's better to use normalization just after we get `axis` value, so all code that's working with it knows about normalized values only

Reply:
Fixed, please take another look.